instrumentation/excon/lib/opentelemetry/instrumentation.rb

```diff
@@ -5,12 +5,12 @@
 # SPDX-License-Identifier: Apache-2.0
 
 module OpenTelemetry
-  # "Instrumentation adapters" are specified by
+  # "Instrumentation instrumentations" are specified by
```

Comment:
This should be edited, and the permalink below is no longer accurate. This comment has been copypasta'd in many of the instrumentation packages. We'll have to track down and update all of them.

Reply:
updated the permalink